.github/actions/setup-eks-cluster/action.yml

```diff
@@ -35,32 +35,12 @@ runs:
           tags:
            usage: "${{ github.repository_owner }}-${{ github.event.repository.name }}"
            owner: "${{ inputs.owner }}"
-
-        managedNodeGroups:
-        - name: ng-amd64
-          instanceTypes:
-           - t3.medium
-          desiredCapacity: 1
-          spot: ${{ inputs.spot }}
-          privateNetworking: true
-          volumeType: "gp3"
-          volumeSize: 10
-          taints:
-           - key: "node.cilium.io/agent-not-ready"
-             value: "true"
-             effect: "NoExecute"
-        - name: ng-arm64
-          instanceTypes:
-           - t4g.medium
-          desiredCapacity: 2
-          spot: ${{ inputs.spot }}
-          privateNetworking: true
-          volumeType: "gp3"
-          volumeSize: 10
-          taints:
-           - key: "node.cilium.io/agent-not-ready"
-             value: "true"
-             effect: "NoExecute"
+                
```

Comment:
The spaces in this line made this WF fail
https://github.com/cilium/cilium/actions/runs/10703955347/job/29675736910?pr=34685